docs/webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
```

Comment:
Why not use the already existing `webpack.config.js` ? That's what I did for Bank and it is working well. This way we have only one source of truth ;)

Reply:
There is no other `webpack.config.js`, is there?

Reply:
There is not, indeed... My bad :)